server/test/resolve.cpp

```diff
@@ -1211,3 +1211,121 @@ TEST_CASE("Resolve by hosted by")
         FAIL(ex.what());
     }
 }
+
+TEST_CASE("Resolve by Group")
+{
+    try {
+        fty::SampleDb db(R"(
+            items:
+                - type : Datacenter
+                  name : datacenter
+                  items:
+                    - type     : Server
+                      name     : srv11
+                      ext-name : srv11
+                - type : Datacenter
+                  name : datacenter1
+                  items:
+                    - type     : Server
+                      name     : srv21
+                      ext-name : srv21
+                - type : Datacenter
+                  name : datacenter2
+                  items:
+                    - type     : Server
+                      name     : serv
+                      ext-name : serv
+            )");
+
+        Group group;
+        group.name          = "ByName";
+        group.rules.groupOp = fty::Group::LogicalOp::And;
+
+        Group group1;
+        group1.name          = "EmptyWithLink";
+        group1.rules.groupOp = fty::Group::LogicalOp::And;
+
+        {
+            auto& var  = group.rules.conditions.append();
+            auto& cond = var.reset<fty::Group::Condition>();
+            cond.field = fty::Group::Fields::Name;
+
+            auto& var1  = group1.rules.conditions.append();
```

Comment:
I know this is my sin too... but try to use meaning variables names, not var, var1, var2... in case if count of this more than 1 then begin mess...